vcluster/configure/vcluster-yaml/control-plane/components/backing-store/etcd/embedded.mdx

```diff
@@ -20,7 +20,8 @@ controlPlane:
   backingStore:
     etcd:
       embedded:
-        ...
+        enabled: false # default value
+        migrateFromDeployedEtcd: false #default value
```

Comment:
I think if this must be placed under "You can migrate from an external etcd instance to the embedded one by configuring `migrateFromDeployedEtcd: true`." Then this should be true. Another option would be to keep this false and then change the line above to be a note either below or above so its clearly a suggestion in addition to what is being presented generally.